taskcluster/kinds/finetune-student/kind.yml

```diff
@@ -108,6 +108,7 @@ tasks:
                     0
                     None
                     None
+                    None
```

Comment:
Thought: I really want to rewrite the training scripts to all use a consistent python training script... but after our big training run.